### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, open 'C:\Users\hmemcpy\.cache\yeoman\sha'

### ISSUE THREAD

**Issue by hmemcpy** [fsprojects/atom-fsharp#98]
I just installed a clean Windows, no Visual Studio yet.
Installed node with chocolatey
Installed yeoman and the F# generator with npm
Installed atom-fsharp.

Attempting to use the generator from either Atom or command line results in the error below.

Looks like I'm missing the `sha` package... should I manually install it?

**Atom Version**: 1.0.2
**System**: Microsoft Windows 10 Pro
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: ENOENT: no such file or directory, open 'C:\Users\hmemcpy\.cache\yeoman\sha'

```
At fs.js:519

Error: ENOENT: no such file or directory, open 'C:\Users\hmemcpy\.cache\yeoman\sha'
    at Error (native)
    at Object.fs.openSync (fs.js:544:18)
    at Object.module.(anonymous function) [as openSync] (ATOM_SHELL_ASAR.js:118:20)
    at Object.fs.writeFileSync (fs.js:1155:15)
    at Object.fs.appendFileSync (fs.js:1199:6)
    at yeoman.generators.Base.extend._saveSHA (C:\Users\hmemcpy\AppData\Roaming\npm\node_modules\generator-fsharp\app\index.js:42:12)
    at yeoman.generators.Base.extend._checkSHA (C:\Users\hmemcpy\AppData\Roaming\npm\node_modules\generator-fsharp\app\index.js:56:15)
    at Request._callback (C:\Users\hmemcpy\AppData\Roaming\npm\node_modules\generator-fsharp\app\index.js:74:17)
    at Request.self.callback (C:\Users\hmemcpy\AppData\Roaming\npm\node_modules\generator-fsharp\node_modules\request\request.js:197:22)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:169:7)
    at Request.<anonymous> (C:\Users\hmemcpy\AppData\Roaming\npm\node_modules\generator-fsharp\node_modules\request\request.js:1050:14)
    at emitOne (events.js:82:20)
    at Request.emit (events.js:166:7)
    at IncomingMessage.<anonymous> (C:\Users\hmemcpy\AppData\Roaming\npm\node_modules\generator-fsharp\node_modules\request\request.js:996:12)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:163:7)
    at _stream_readable.js:891:16
    at process._tickCallback (node.js:357:13)
```

### Commands

```
     -2:32 yo:yeoman (atom-pane.pane.active)
     -2:25.3.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -2:24.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:09.9.0 yo:yeoman (atom-pane.pane.active)
     -2:06.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:05.5.0 yo:yeoman (atom-pane.pane.active)
     -0:03.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{}
```

### Installed Packages

```coffee
# User
atom-fsharp, v0.8.0
atom-yeoman, v0.3.11
linter, v1.2.4
omnisharp-atom, v0.20.0

# Dev
No dev packages
```

**Comment by Krzysztof-Cieslak** [comment]
It's problem connected with F# generator - reported in https://github.com/fsprojects/generator-fsharp/issues/10 - the discussion contains some workaround but i still haven't fixed bug.

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed in `0.4.3` version of generator
